axelrod/strategies/worseandworse.py

```diff
@@ -0,0 +1,35 @@
+from axelrod import Actions, Player
+from random import randint
+
+class WorseAndWorse (Player):
+    """
+    Defects with probability of 'current turn / total no. of turns'. Therefore
+    it is more and more likely to defect as the round goes on.
+
+    Names:
+        - worse_and_worse: [PRISON1998]
+
+    """
+
+    name = 'Worse and worse'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': set(['length']),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def strategy(self, opponent):
+        current_round = len(self.history) + 1
+        expected_length = self.match_attributes['length']
+        try:
+            if randint(0, expected_length) < (current_round):
+                return Actions.D
+            return Actions.C
+        except:
```

Comment:
@drvinceknight would you prefer it to be an if/else statement? The reason why travis is failing is because random.seed(1) and random.seed(2) is different for python 2.7 and python 3